### ISSUE TITLE
[v1.13beta1] multiple select_item event (repeated from #186) 

### ISSUE THREAD

**Issue by prabapro** [duracelltomi/gtm4wp#190]
Version: v1.13beta1

Hi @duracelltomi,

As reported on #186, event select_item pushes multiple times on `v1.13beta1` as well.

_PS: tested this on woo.gtm4wp.dev & the issue was not there. But the site uses `version 1.13.20210501`, so guess the issue is back with the latest beta_.

Thanks,
Praba.

**Comment by duracelltomi** [comment]
Thanks for the report!

I synced the code on my dev site to make sure it is identical with 1.13 beta1 and this part of the code is the same as in the release files. Therefore if you see this working on my dev site and not working on your site then this seems to be a specific issue.

Could you post an URL of your site where this is not working as expected?

**Comment by prabapro** [comment]
Hi Thomas,

Thanks for your reply.

Did injected the container on `woo.gtm4wp.dev` & didn't notice this issue. But it's there on my sandbox site (something similar to your sandbox with StoreFront theme).

**URL**
```
https//gtmsandbox.codechilli.lk
```

```
StoreFront 3.6.0
WordPress 5.7.2
WooCommerce 5.3.0
PHP 7.4.3
```

**Preview Mode**
```
https://tagassistant.google.com/#/?id=GTM-NQDK8NJ&url=https%3A%2F%2Fgtmsandbox.codechilli.lk%2Fproduct%2Fcap%2F&source=TAG_MANAGER&gtm_auth=pAlmeTeu-je9Qinmp73eHA&gtm_preview=env-16
```

Thanks once again :)

Praba.

**Comment by duracelltomi** [comment]
OK, that is weird, could you please apply this patch?

https://github.com/duracelltomi/gtm4wp/commit/067d24c75daf3db71bf28650d19a0f58d153e73d

**Comment by prabapro** [comment]
Hi Thomas,

Applied the fix 067d24c & it seems fixed now. Thank you so much.  Much appreciated :)

Thanks,
Praba.
